.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+defaults: &defaults
+  docker:
+    - image: quay.io/pantheon-public/php-ci:latest
+  working_directory: ~/work/terminus_rsync_plugin
+  environment:
+    BASH_ENV: ~/.bashrc
+    TZ: "/usr/share/zoneinfo/America/Los_Angeles"
+    TERM: dumb
+
+version: 2
+jobs:
+    test:
+        <<: *defaults
+        steps:
+            - checkout
+            - run:
+                name: Set up environment
+                command: ./.circleci/set-up-globals.sh
+            - run:
+                name: Dependencies
+                command: composer install
```

Comment:
Note that we are running `composer install` here even though we do not have any dependencies of our own because this project is intended to be an example that other plugin authors can duplicate to port their Terminus plugins to Circle 2.